frontend/src/index.css

```
@@ -43,12 +43,14 @@
 @layer base {
   body {
     @apply h-dvh;
-    font-family: "Manrope", sans-serif;
+    font-family: var(--app-font-family);
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)